### DISCUSSION TITLE
Could argo deployment have multiple replicas

### DISCUSSION THREAD

**Discussion by journey-wang** [argoproj/argo-workflows#8675] [Q&A] [answered]
Hi,

Could workflow-controller and argo-server deployment scales to multiple replicas, not 1.

---

**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritise the issues with the most 👍.

**Comment by terrytangyuan** [comment, marked as answer]
See this page for relevant information: https://argoproj.github.io/argo-workflows/scaling/
